app/parse.py

```diff
@@ -8,8 +16,70 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
```

Comment:
QUOTE_FIELDS will be calculated every time someone imports your module, place it in a function or program start-point

Reply:
I am with you here